script.js

```diff
@@ -434,60 +450,108 @@ function placeCard(event) {
   // remove played card from hand
   source.remove();
   // switch turns after card is played
+  evaluateBattles(dest);
   flipCards();
   checkForEmptyCells();
 }
 
-function checkForEmptyCells() {
-  cellElem.forEach(cell => {
-    if (cell.dataset.played !== 'true') {
-      cell.addEventListener('click', placeCard);
-    } else if (cell.dataset.played == 'true') {
-      cell.removeEventListener('click', placeCard);
-    }
-  });
-  switchTurns();
-}
-
 function switchTurns() {
-  if (isPlayerTurn) {
-    isPlayerTurn = false;
-  } else if (!isPlayerTurn) {
-    isPlayerTurn = true;
+  if (isBlueTurn) {
+    isBlueTurn = false;
+  } else if (!isBlueTurn) {
+    isBlueTurn = true;
   }
 }
 
-// // parse "A" as integer
-// function parseRankAsInt(rank) {
-//     console.log("Parsing Rank...");
-// }
-
 // // evaluate competiting values
-// function evaluateValues(selected, dest, playedCards) {
-//     console.log("Evaluating Values...");
-// }
-
-// // check for winner
-// function checkForWin(table) {
-//     console.log("Checking For Winner...");
-// }
+function evaluateBattles(dest) {
+  // turn cell id number of target cell into integer
+  let destCell = parseInt(dest.dataset.cell);
+  // store starting ownership for repeat use in evaluations
+  let owner = dest.dataset.owner;
+  // turn all grid cells into an array for location purposes
+  let cellArray = Array.prototype.slice.call(cellElem);
+  // map placed cards html to grid location
+  let placedCard = [...dest.children].map(cont => cont.innerHTML);
+  // remove img from array
+  placedCard.shift();
+  // loop through cells in grid
+  for (let cell in cellArray) {
+    cell = cellArray[cell];
+    // set value directions to variable based on destination cell for placed card
+    // use destination cell with cellArray to determine competing cards location
+    // !== '' checks for empty space to skip evaluation
+    // cellArray index is adjusted by -1 in comparision to value directions since cellArray begins at 0, not 1;
+    let up = (destCell - 3);
+    let left = (destCell - 1);
+    let right = (destCell + 1);
+    let down = (destCell + 3);
+    if (cell.dataset.cell == up && cell.innerText !== '') {
```

Comment:
Another solid idea. Should eat up a chunk of characters.
Will go with something like `touchingCell`.
With that do you think `destCell` is fine or should it be more defined as `destinationCell`?

Reply:
`destCell` seems fine to me!

Reply:
So after updating the script with these variables there was a repeat of an issue I encountered earlier.

If you'll notice I did something similar to this: `let owner = dest.dataset.owner;`

The issue arises during evaluation where once the variable is set, it will remain as such for the subsequent evaluations. This causes an undesirable result when the card evaluations are involving a matching owner or there are multiple results from the evaluation.  

This may be tied directly to how the function is written. I'll fiddle with it after refactoring.